src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.sdk;
+
+import java.io.IOException;
+
+import org.apache.http.HttpHost;
+import org.apache.http.auth.AuthScope;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.CredentialsProvider;
+import org.apache.http.conn.ssl.NoopHostnameVerifier;
+import org.apache.http.impl.client.BasicCredentialsProvider;
+import org.apache.http.ssl.SSLContextBuilder;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.client.RestClient;
+import org.opensearch.client.RestClientBuilder;
+import org.opensearch.client.json.jackson.JacksonJsonpMapper;
+import org.opensearch.client.opensearch.OpenSearchClient;
+import org.opensearch.client.opensearch.indices.CreateIndexRequest;
+import org.opensearch.client.opensearch.indices.CreateIndexResponse;
+import org.opensearch.client.transport.OpenSearchTransport;
+import org.opensearch.client.transport.rest_client.RestClientTransport;
+
+/**
+ * This class creates a Client for SDK to make requests to OpenSearch
+ */
+public class SDKClient {
+    private final Logger logger = LogManager.getLogger(SDKClient.class);
+    private OpenSearchClient client;
+
+    /**
+     * Creates client for SDK
+     * @throws IOException if client failed
+     */
+    public void createClient() throws IOException {
+        String endpoint = "localhost";
+        String username = "admin";
+        String password = "admin";
```

Comment:
I understand this is we suggest to implement. 
To make this iterative, I would remove everything security for now and have a bare bones client i.e HTTP and everything insecure. 

Long term, depending on how security is going to fan out we could explore options:
- As @dbwiddis we might have token based access. 
- If we are looking for stop gap solution, I would get this information from our `.yml` config file, very likely load certificates and present them to OpenSearch.

@dbwiddis @peternied @owaiskazi19 thoughts?

Reply:
Kerberos is actual AuthN so its a great option - however I have never set it up with OpenSearch so I don't know how burdensome it will be to configure/use.

I would recommend using connection strings the work with AuthN/Z disabled (good for setting up quickly), then you can embed within the connection string basic auth credentials - this is the standard way to setup an OpenSearch cluster quickly (like pulling in a docker image)

Reply:
Sounds good. Removed security from the SDKClient and will be addressed [here](https://github.com/opensearch-project/opensearch-sdk/issues/90)